package/android/src/main/java/com/mrousavy/camera/extensions/Rect+zoomed.kt

```diff
@@ -3,12 +3,11 @@ package com.mrousavy.camera.extensions
 import android.graphics.Rect
 
 fun Rect.zoomed(zoomFactor: Float): Rect {
-  val height = bottom - top
-  val width = right - left
-
-  val left = this.left + (width / zoomFactor / 2)
-  val top = this.top + (height / zoomFactor / 2)
-  val right = this.right - (width / zoomFactor / 2)
-  val bottom = this.bottom - (height / zoomFactor / 2)
-  return Rect(left.toInt(), top.toInt(), right.toInt(), bottom.toInt())
+  val dx = (width() / zoomFactor / 2).toInt()
+  val dy = (height() / zoomFactor / 2).toInt()
+  val left = centerX() - dx
+  val top = centerY() - dy
+  val right = centerX() + dx
+  val bottom = centerY() + dy
+  return Rect(left, top, right, bottom)
```

Comment:
What's the difference here? Does that still work for all cases?

Reply:
The old code zoomed-out more the larger the zoomFactor was, this fixes this. It should work for the same cases, tested it on my device.